### ISSUE TITLE
Add profile settings page for user

### ISSUE THREAD

**Issue by xndrdev** [JustServerless/discuss#27]
A profile settings page, that the user can update his email, password and some personal information.

**Comment by pmuens** [comment]
Good idea! Should we allow the user to update his username? I wouldn't vote for that as it makes validations hard.

**Comment by xndrdev** [comment]
yeah, you're right! only should update the password.
